src/power-bi/storage/Shared.Dataset/definition/expressions.tmdl

```diff
@@ -475,7 +475,7 @@ expression 'Experimental: Add Missing Prices' = false meta [IsParameterQuery=tru
 
 	annotation PBI_NavigationStepName = Navigation
 
-	annotation PBI_ResultType = Logical
+	annotation PBI_ResultType = Exception
```

Comment:
Can you make sure these queries fully load in the Transform Queries window? If you stop it before it's done, then it shows "Exception". All you need to do is make sure it fully loads. We don't want them showing errors when customers open them because we didn't let it finish loading. Or even worse would be if this is indicating a real problem.